sphinx/ext/autodoc/directive.py

```diff
@@ -84,7 +84,12 @@ def process_documenter_options(documenter: "Type[Documenter]", config: Config, o
         else:
             negated = options.pop('no-' + name, True) is None
             if name in config.autodoc_default_options and not negated:
-                options[name] = config.autodoc_default_options[name]
+                if name == "exclude-members":
```

Comment:
I think all options taking a list of member names should be extended at runtime, not only `exclude-members`. For example,`private-members`, `undoc-members`, `special-members` and so on.
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_default_options

Reply:
Added possibility to extend options `members`,`private-members`, `special-members` and `exclude-members`, other options have a priority over `autodoc_default_options`